### PR TITLE
Tweak release.sh

### PR DESCRIPTION
**Pull request by paulo-ferraz-oliveira**
Just a bit 😄

### CODE REVIEW

**Comment by starbelly** [comment]
LGTM, let me test locally, then I'll merge

File: release.sh [hunk 1]
```diff
@@ -1,29 +1,29 @@
...
+            [Nn][Oo]) echo "Ok, bye bye!"; exit ;;
+            * ) echo "Please answer 'yes' or 'no'." ;;
         esac
     done
 }
 
-if [ "${OTP_VER}" != "24" ]; then
-     echo "You are using Erlang/OTP ${OTP_VER_STR}"
-     echo "However, ex_doc must be built using OTP == 24 for compatability. Giving up..."
+if [ $((OTP_VER)) -lt 24 ]; then
```
Comment (starbelly) [inline release.sh:17]: This doesn't play well with my bash shell, sadly.

Comment (starbelly) [inline release.sh:17]:
```suggestion
if [ $OTP_VER -ne 24 ]; then
```

Comment (starbelly) [reply release.sh:17]: ^ has to be exactly 24. If we publish with OTP 25 or 26, it will break for others.

Comment (paulo-ferraz-oliveira) [reply release.sh:17]: Ok.

Comment (paulo-ferraz-oliveira) [reply release.sh:17]: Hm... what version of `bash` are you using? The change I propose is, as far as I know, standard: https://www.gnu.org/software/bash/manual/html_node/Arithmetic-Expansion.html

File: release.sh [hunk 2]
```diff
@@ -1,29 +1,29 @@
...
+            [Nn][Oo]) echo "Ok, bye bye!"; exit ;;
+            * ) echo "Please answer 'yes' or 'no'." ;;
         esac
     done
 }
 
 if [ "${OTP_VER}" != "24" ]; then
-     echo "You are using Erlang/OTP ${OTP_VER_STR}"
-     echo "However, ex_doc must be built using OTP == 24 for compatability. Giving up..."
+     echo "You are using Erlang/OTP ${OTP_VER}."
```
Comment (paulo-ferraz-oliveira) [inline release.sh:18]: This is now the only relevant change.

File: release.sh (hunk 1 above)
Comment (starbelly) [reply release.sh]: GNU bash, version 3.2.57(1)-release (arm64-apple-darwin22)

In retrospec, it wasn't the syntax, it was the `-lt 24` bit (i.e., I was testing it by running 26, but you `-lt 24`, so it didn't blow up as it should, I simply glazed over that detail at the time.

**Pull request merged**